### DISCUSSION TITLE
Podcasts sobre front-end

### DISCUSSION THREAD

**Discussion by leoamrl** [frontendbr/forum#1108] [Dicas]
Eu gostava bastante de ouvir podcasts durante o trabalho mas acabei parando por não encontrar conteúdo bacana e os que tinham e eram legais, acabaram sendo descontinuados ou não tinham muito material.

Gostaria de indicações de podcast/canais no youtube sobre desenvolvimento com um conteúdo bacana e se possível com atualização periódica, só pra ter algo legalzinho pra ouvir durante as horas codando.

Se possível um material gringo, seria show.

**Comment by wmsbill** [comment]
http://frontendhappyhour.com/

https://dev.to/basecspodcast

**Comment by vituhernandez** [comment]
Hipsters Tech tem um conteúdo bacana mano, recomendo

Em seg, 4 de jun de 2018 às 10:58, Leo Amaral <notifications@github.com>
escreveu:

> Eu gostava bastante de ouvir podcasts durante o trabalho mas acabei
> parando por não encontrar conteúdo bacana e os que tinham e eram legais,
> acabaram sendo descontinuados ou não tinham muito material.
>
> Gostaria de indicações de podcast/canais no youtube sobre desenvolvimento
> com um conteúdo bacana e se possível com atualização periódica, só pra ter
> algo legalzinho pra ouvir durante as horas codando.
>
> Se possível um material gringo, seria show.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/frontendbr/forum/issues/1108>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AhTROpKw7XCi7YyIGpn8gQLkvWE4aSz2ks5t5T0ZgaJpZM4UZI4h>
> .
>
-- 


-- 
att, Victor Hernandez - BUBB Agência Web
16 3341 8797

**Comment by klarkc** [comment]
https://devnaestrada.com.br/ Tinha q ser a primeira indicação, tão vacilando po? :rofl:

**Comment by Sergioamjr** [comment]
Tem esses gringos
https://syntax.fm/
https://changelog.com/reactpodcast
Coleção de vários https://devchat.tv/
